cmd/crawl.go

```diff
@@ -3,23 +3,36 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+	"log"
+	"net/url"
 
 	"github.com/OpenCHAMI/magellan/pkg/crawler"
 	"github.com/spf13/cobra"
 )
 
 var crawlCmd = &cobra.Command{
-	Use:   "crawl",
+	Use:   "crawl [uri]",
 	Short: "Crawl a single BMC for inventory information",
```

Comment:
Perhaps we could add a long description below this that describes what `uri` is. I don't think there is a standard way to do this in Cobra yet.

Reply:
I want to second this. I think we also want to include that `uri` is the host and not the Redfish root endpoint as mentioned in [the issue](https://github.com/OpenCHAMI/magellan/issues/36#issuecomment-2200932275).